cmd/ipfs-cluster-service/main.go

```diff
@@ -272,6 +273,10 @@ the peer IDs in the given multiaddresses.
 					Name:  "force, f",
 					Usage: "overwrite configuration without prompting",
 				},
+				cli.BoolFlag{
+					Name:  "randomports",
+					Usage: "listen on random available ports, instead of the default values",
```

Comment:
`configure random ports to listen on instead of defaults`

Reply:
(we are not listening on random ports, we are just generating the configuration by selecting a random port).